apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -203,18 +207,26 @@ defmodule AndiWeb.EditLiveView do
 
     Process.flag(:trap_exit, true)
 
+    crontab =
```

Comment:
Is there any way this could be put inside the live component itself? I'm guessing this is just for making it work with the form and we unmunge it on the way out?

Reply:
Or maybe just find a way to make it more clear that this conversion is not permanent?

Reply:
Yeah, absolutely. Really no reason why it's there